src/DynamoCLI/DynamoCLI.csproj

```diff
@@ -69,6 +70,10 @@
       <Name>DynamoApplications</Name>
       <Private>False</Private>
     </ProjectReference>
+    <ProjectReference Include="..\DynamoCoreWpf\DynamoCoreWpf.csproj">
```

Comment:
one downside of this is that now the CLI will not compile or run on Mac or linux.

Reply:
note the failing mono build, if we want to merge this we should update the mono solution as well... I wonder if we should add a new project for WPF CLI - or maybe your initial workaround is the way to go. Does having a Mac/linux CLI have much utility?

Reply:
Yes, I noticed that the mono build now is failing. It doesn't looks bad from a performance standpoint. Executing my test graph multiple times is now about 6s and profiling says that we lost another 0.4s and the new average is now 16.2s. But I have some problems (crashes) on bigger graphs that I am looking into now.

Reply:
Using the DefaultWatch3DViewModel for the CLI fixed all instability issues I was seeing.

Reply:
I guess we have three options:
1. Use the original proposal and not use DynamoViewModel for the CLI.
2. Use the current proposal and use DynamoViewModel for the CLI. Remove CLI from the mono build.
3. Create a new project, DynamoWpfCLI, that uses DynamoViewModel. It is indentical to DynamoCLI but do support the geometry option.

Reply:
@QilongTang @ramramps what are your thoughts on the above options?

My feeling is number 3 even though it is more code, bigger solution, and a bit slower, we still have the option in the future to remove the regular CLI if it gets no use.

If this solution proves to slow for @sm6srw 's needs then we can use 1. and modify deserialization.

Reply:
depends on where DynamoCLI will be used. Using the option 3, it is strictly windows. 
I would still prefer CLI to work on multiple environments. By removing the regular CLI, we might send the signal that dynamo computation is windows specific?

Reply:
options 3 is the introduction of a new project, something like dynamoCLIWPF - so we keep the old CLI for now and we can run CLI in any environment.

Reply:
I don't have a strong opinion about that. how many users use CLI at the first place for computation? Does CLI proves to be a useful project for external devs?  
 may be we should revisit the parameters we have for node compute vs node view, and how each parameter helps in the computation. 

If option 3 is the fastest to get this working, then it is fine for now.